_posts/2023-02-28-admin-panels-for-index-operations.md

```diff
@@ -0,0 +1,138 @@
+
+<img src="/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/alias-creation.png" alt="create an alias"/>{: .img-fluid }
+
+### 5. Simplify the flow for the reindex operation.
+
+You can easily select the source and destination indexes, aliases, or data streams from the dropdown menu. Additionally, we provide an ad-hoc destination index creation flow that allows you to import settings and mappings directly from the source, making the process more convenient.
+
+![Image: Reindex page]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/reindex.png){:.img-fluid }
+
+### 6. Shrink index operation
+
+Shrink index operation is used to shrink an existing index into a new index with fewer primary shards.
+
+When you want to shrink an index, go to the indices pages, select one index, click the `Shrink` action in the actions menu then you will enter the shrink index page. Please notice that only one index can be shrunk at once and data stream indices are not supported to do shrink, so if multiple indices are selected or a data stream backing index is seletecd in the indices pages, the `Shrink` option is disabled. 
+
+![Image: Shrink-action]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-action.jpg){:.img-fluid }
+
+Before shrinking, please make sure that the index you want to shrink is not in these states:
+
+* the index's health status is red.
+* the index has only one primary shard.
+
+If the index is in one of the states above, some error messages will be shown in the shrink index page while the shrink button is disabled.
+
+![Image: Shrink-red]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-red.jpg){:.img-fluid }
+
+![Image: Shrink-one-shard]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-one-shard.jpg){:.img-fluid }
+
+Shrink index operation has some prerequisites, if the source index does not meet the conditions, some error messages will be shown in the shrink index page.
+
+One condition is that the source index must block write operations, i.e. the `index.blocks.write` setting in the source index is `true`, if the source index is not set to block write operations, then you can click the `Block write operations` button to set the `index.blocks.write` setting to `true`.
+
+![Image: Shrink-block-write]({{site.baseurl}}/assets/media/blog-images/2023-02-28-admin-panels-for-index-operations/shrink-block-write.jpg){:.img-fluid }
+
+The second condition is that the source index must be open, when the source index is closed, the shrink operation will fail, so you can click the `Open` button to open the index, this may take additional time to complete and the index will be in red health status while opening.
```

Comment:
```suggestion
Secondly, the source index must be open. When the source index is closed, the shrink operation will fail. You can click the `Open` button to open the index. This may take additional time to complete and the index will be in the red health status while opening.
```